l1-contracts/contracts/common/Dependencies.sol

```diff
@@ -1,6 +1,6 @@
 // SPDX-License-Identifier: MIT
 
-pragma solidity 0.8.24;
+pragma solidity ^0.8.20;
```

Comment:
Why it is ^0.8.20, and not let's say ^0.8.13? Do we use some features from 0.8.20 and higher?

Reply:
i figured this was the easiest way to avoid potential breaking changes esp given that most code bases that already needed to integrate would be on pragma `0.8.24`